tests/karma/osu-core.spec.ts

```diff
@@ -26,7 +26,8 @@ describe('OsuCore user:update subscriber testing thing', () => {
       username: 'foo',
     };
 
-    const user = User.fromJson(json);
+    const user = new User(json.id);
+    user.updateWithJson(json);
```

Comment:
just noticed this but the listener actually expects json...

Reply:
... 🤔 

this test is weird (hardcoded `1` and trying to shallow compare User instance)